Userland/Utilities/image.cpp

```diff
@@ -125,8 +125,20 @@ static ErrorOr<OwnPtr<Core::MappedFile>> convert_image_profile(LoadedImage& imag
 
 static ErrorOr<void> save_image(LoadedImage& image, StringView out_path, bool ppm_ascii, u8 jpeg_quality)
 {
-    if (!image.bitmap.has<RefPtr<Gfx::Bitmap>>())
-        return Error::from_string_view("Can't save CMYK bitmaps yet, convert to RGB first with --convert-to-color-profile"sv);
+    if (image.bitmap.has<RefPtr<Gfx::CMYKBitmap>>()) {
+        auto& cmyk_frame = image.bitmap.get<RefPtr<Gfx::CMYKBitmap>>();
+
+        auto output_stream = TRY(Core::File::open(out_path, Core::File::OpenMode::Write));
+        auto buffered_stream = TRY(Core::OutputBufferedFile::create(move(output_stream)));
```

Comment:
These two can be moved out of the `if`.

Reply:
I had it like this so we don't create the outfile if we then can't write to it. But that was a preexisting problem too, so I added a commit to fix that more thoroughly.